### PR TITLE
Build touch instead of touch-pad and touch-phone

### LINKED ISSUE #1005: Build touch instead of touch-pad and touch-phone

### PR DESCRIPTION
**Pull request by tadatuta**
closes #1005 

cc @blond @gela-d

### CODE REVIEW

**Comment by blond** [comment]
:+1:

**Comment by gela-d** [comment]
do not forget about `bem-data-source` - https://github.com/bem-site/bem-data-source/blob/master/declarations/bem-core.js

File: .enb/make.js [hunk 1]
```diff
@@ -24,15 +24,14 @@ var LIB_NAME = 'bem-core',
     copyFile = require('enb/techs/file-copy'),
     mergeFiles = require('enb/techs/file-merge'),
     borschik = require('enb-borschik/techs/borschik'),
-    PLATFORMS = {
+    SETS = {
         'common' : ['common'],
```
Comment (gela-d) [inline .enb/make.js:28]: Are we need this level?

File: package.json [hunk 2]
```diff
@@ -46,8 +46,8 @@
   },
   "scripts": {
     "start": "magic server",
-    "dist": "npm run deps && magic make dist/desktop dist/touch-pad dist/touch-phone",
-    "build": "npm run deps && magic make desktop.examples desktop.tests desktop.docs touch-pad.examples touch-pad.tests touch-pad.docs touch-phone.examples touch-phone.tests touch-phone.docs *.bundles/*",
+    "dist": "npm run deps && magic make dist/desktop dist/touch",
+    "build": "npm run deps && magic make desktop.examples desktop.tests desktop.docs touch.examples touch.tests touch.docs touch.examples touch.tests touch.docs *.bundles/*",
```
Comment (gela-d) [inline package.json]: duplicate commands
`touch.examples touch.tests touch.docs touch.examples touch.tests touch.docs`

File: .enb/make.js (hunk 1 above)
Comment (tadatuta) [inline .enb/make.js:28]: we have `common.bundles`

**Comment by gela-d** [comment]
:+1:

File: package.json [hunk 3]
```diff
@@ -46,8 +46,8 @@
   },
   "scripts": {
     "start": "magic server",
-    "dist": "npm run deps && magic make dist/desktop dist/touch-pad dist/touch-phone",
-    "build": "npm run deps && magic make desktop.examples desktop.tests desktop.docs touch-pad.examples touch-pad.tests touch-pad.docs touch-phone.examples touch-phone.tests touch-phone.docs *.bundles/*",
+    "dist": "npm run deps && magic make dist/desktop dist/touch",
+    "build": "npm run deps && magic make desktop.examples desktop.tests desktop.docs touch.examples touch.tests touch.docs *.bundles/*",
```
Comment (sipayRT) [inline package.json:50]: why should we build `tests`?

> `*.bundles/*`

what bundles you want to build here?
Can we write this command more compact? For example, `npm run deps && magic run examples docs && magic make *.bundles/*`

File: package.json (hunk 2 above)
Comment (blond) [inline package.json]: @sipayRT +1

File: package.json (hunk 3 above)
Comment (tadatuta) [inline package.json:50]: decided to do it as a separate [issue](https://github.com/bem/bem-core/issues/1010)

**Pull request merged**